rio_pansharpen/scripts/cli.py

```diff
@@ -7,8 +7,10 @@
 @click.command('pansharpen')
 @click.argument('src_paths', type=click.Path(exists=True), nargs=-1)
 @click.argument('dst_path', type=click.Path(exists=False), nargs=1)
-@click.option('--dst-dtype', type=click.Choice(['uint16', 'uint8']),
-              default='uint8')
+@click.option('--dst-dtype',
+              type=click.Choice(['int16', 'int8', 'uint16', 'uint8']),
+              default='uint16')
+@click.option('--ndv', default=None)
```

Comment:
@virginiayung I'm not worried about the dst-dtype in #17 - the bigger concern is the assumption below that the *src-dtype* is uint16